src/Evino/Nebbiolo/DataTransfer/Base/BaseDataTransfer.php

```diff
@@ -66,11 +66,18 @@ public function jsonSerialize()
     /**
      * Converts DTO to JSON string
      *
+     * @param bool $includeNullValues
      * @return string
      */
-    public function toJson()
+    public function toJson($includeNullValues = false)
     {
-        return json_encode($this);
+        $encoded = json_encode($this);
+
+        if (!$includeNullValues) {
+            $encoded = preg_replace('/,\s*"[^"]+":null|"[^"]+":null,?/', '', $encoded);
+        }
+
+        return $encoded;
```

Comment:
no love for ternary?

Reply:
I like ternaries when the expression looks simple and clean and it wouldn't be the case here.